src/main/scala/org/zalando/spearheads/innkeeper/oauth/OAuthDomain.scala

```diff
@@ -2,18 +2,20 @@ package org.zalando.spearheads.innkeeper.oauth
 
 import com.google.inject.{ Inject, Singleton }
 import com.typesafe.config.Config
+import scala.collection.immutable.List
+import scala.collection.JavaConversions._
 
 /**
  * @author dpersa
  */
-case class Scope(val name: String) extends AnyVal
+case class Scope(val scopeNames: List[String]) extends AnyVal
```

Comment:
Why `List`? Seems to me that a `Set` is a better fit. Or at least `Seq`. But List is overly specific